R/stage_nldas_ts.R

```diff
@@ -54,7 +54,7 @@ stage_nldas_ts <- function(sites, var, times, folder = tempdir(), verbose = FALS
     
     site_data <- select(site_data, -units) %>%
       setNames(c('DateTime',var)) %>%
-      u(c("UTC", units))
+      u(c(NA, units))
```

Comment:
read_ts turns units='UTC' into tz='UTC', so I thought write_ts should do the opposite, tz='UTC' to units='UTC'. So I moved that functionality into write_ts. This also has the benefit of not requiring every stage_xxx_ts function to remember to add 'UTC' to the units, though I realize this is a tiny benefit.